.circleci/config.yml

```diff
@@ -33,6 +33,21 @@ jobs:
         environment:
           PYTEST_FILTER: notebooks
 
+  format-checker:
+    docker:
+      - image: awav/tensorflow:2.0-preview
```

Comment:
when are we upgrading this to awav/tensorflow:2.1 ?

Reply:
The 2.0-preview doesn't mean it uses 2.0 version. Docker image doesn't have pre-installed tensorflow, we install things via pip. I will address the image name in different PR.